src/pages/homepage/HomePage.js

```diff
@@ -1,10 +1,14 @@
-import { Chip } from '@mui/material'
-import React from 'react'
-import { Sidebar } from '../../components/sidebar/Sidebar'
-import './homepage.css'
-import { VideoCard } from '../../components/videocard/VideoCard'
+import { Chip } from '@mui/material';
+import React from 'react';
+import { Sidebar } from '../../components/sidebar/Sidebar';
+import './homepage.css';
+import { VideoCard } from '../../components/videocard/VideoCard';
+import { useVideoList } from "../../context/videoContext";
+
 export const HomePage = () => {
+const {videoList} = useVideoList();
 
+console.log(videoList)
   const handleClick = () =>{
```

Comment:
If possible remove the console statement. I think you didn't use any statements inside handleClick. remove arrow function.

Reply:
thanks @santhoshpemmaka  , good pick 
okay I will remove it